src/lib.rs

```diff
@@ -135,6 +140,7 @@ impl AsRef<Cache> for CacheAndHttp {
 #[cfg(feature = "client")]
 impl AsRef<Http> for CacheAndHttp {
     fn as_ref(&self) -> &Http {
+        let _ = 12_u64 as u8;
```

Comment:
Missed a spot ;)

Reply:
Oh crap I wanted to verify my "else the blanket warnings would override the individual allow's" claim but forgot to take the test code out :sweat_smile: Thanks for your rigorous reviewing